lib/engine/train.rb

```diff
@@ -10,7 +10,7 @@ class Train
     attr_accessor :obsolete, :operated, :events
     attr_reader :available_on, :name, :distance, :discount, :obsolete_on,
                 :rusts_on, :rusted, :sym, :variant, :variants
-    attr_writer :buyable
+    attr_writer :buyable, :index
```

Comment:
what's the point of index?

Reply:
When adding the new 2 train I do that by cloning another 2 train, and then I set index so it wont collide with another 2 train.